src/3rd_party_bundle_add.c

```diff
@@ -114,22 +114,10 @@ enum swupd_code third_party_bundle_add_main(int argc, char **argv)
 {
 	struct list *bundles = NULL;
 	enum swupd_code ret_code = SWUPD_OK;
-
-	/*
-	 * Steps for 3rd-party bundle-add:
-	 *
-	 *  1) load_manifests
-	 *  2) download_packs
-	 *  3) extract_packs
-	 *  4) validate_fullfiles
-	 *  5) download_fullfiles
-	 *  6) extract_fullfiles
-	 *  7) install_files
-	 */
-
-	const int steps_in_bundleadd = 7;
+	const int steps_in_bundleadd = 8;
```

Comment:
bundle_add

Reply:
did we decide to use them as a single_word?

Reply:
here it does not matter, is just a variable name